StepDefinitions/MyProfileLanguagesStep.cs

```diff
@@ -0,0 +1,125 @@
+using NUnit.Framework;
+using ProjectMars.Pages;
+using ProjectMars.Utilities.Hooks;
+using TechTalk.SpecFlow;
+
+namespace ProjectMars.StepDefinitions
+{
+    /// <summary>
+    /// Step definition for language functionality in my profile.
+    /// </summary>
+    [Binding]
+    public class MyProfileLanguagesStep : Hooks
+    {
+        SignInPage signInPageObj = new SignInPage();
+        ProfilePageLanguages profilePageObj = new ProfilePageLanguages();
+
+        [Given(@"User logs into Mars application")]
+        public void GivenUserLogsIntoMarsApplication()
+        {
+            signInPageObj.SignInToApplication(driver);
+        }
+
+        [When(@"User adds '([^']*)' language with '([^']*)' proficiency level into the known languages list")]
+        public void WhenUserAddsLanguageWithProficiencyLevelIntoTheKnownLanguagesList(string language, string proficiencyLevel)
+        {
+            profilePageObj.AddLanguageWithProficiencyLevel(driver, language, proficiencyLevel);
+        }
+
+        [Then(@"The '([^']*)' language with '([^']*)' proficiency level should appear in the known languages list")]
+        public void ThenTheLanguageWithProficiencyLevelShouldAppearInTheKnownLanguagesList(string language, string proficiencyLevel)
+        {
+            if (profilePageObj.VerifyTheAddedLanguage(driver, language, proficiencyLevel))
```

Comment:
please do not pass the driver in the action methodprofilePageObj.VerifyTheAddedLanguage(driver, language, proficiencyLevel)

Reply:
please fix this for all action methods